### DISCUSSION TITLE
Will v2ray support SNI Shunt in VLESS Fallbacks?

### DISCUSSION THREAD

**Discussion by wintertee** [v2fly/v2ray-core#1078] [Ideas]
This is a feature introduced in XRay XTLS/Xray-core#141 which is very convient for a multi-domain server.
Related issue: #419

**Comment by xiaokangwang** [comment]
You can use nginx to achieve this right now. This might be supported in future.

**Reply by wintertee** [reply]
OK. Thank you!
